README.md

```diff
@@ -1,30 +1,52 @@
-# Sourcegraph extension for Phabricator
+# Sourcegraph's Phabricator integration
 
 Get code intelligence on Phabricator with Sourcegraph.
 
 Check out our [docs site](https://about.sourcegraph.com/docs/features/phabricator-extension) for more information.
 
 ## [Installation](https://https://about.sourcegraph.com/docs/features/phabricator-extension#option-a-single-installation-script)
 
-1. To install Sourcegraph on your Phabricator instance run `./install.sh <path/to/phabricator/root> <sourcegraph url>` inside your Phabricator instance.
-2. Restart Phabricator instance.
+1. Clone this repository into a sourcegraph folder located in your Phabricator extension directory.
 
-Requires PHP 5.5 or newer and `php-tokenizer` (a default PHP extension).
+```
+git clone -b v1.0 git@github.com:sourcegraph/phabricator-extension-install.git path/to/phabricator/src/extensions/sourcegraph
+```
 
-*You may delete this repository after installation*
+2. Regnerate your dependencies and packaging maps with `./bin/celerity map`. See [how to regenerate static resources](https://secure.phabricator.com/book/phabdev/article/celerity/).
 
-## [Manual Installation](https://about.sourcegraph.com/docs/features/phabricator-extension#option-b-manual-installation)
+3. [Restart Phabricator](https://secure.phabricator.com/book/phabricator/article/restarting/).
 
-1. To manually install Sourcegraph view our [documentation](https://about.sourcegraph.com/docs/features/phabricator-extension)
+## Quickstart
 
-## [Uninstall](https://about.sourcegraph.com/docs/features/phabricator-extension#uninstall)
+Sourcegraph's Phabricator integration adds Sourcegraph code intelligence and search to Phabricator diffs and code files, so you get go-to-definition, find-references, hover tooltips, and code search embedded natively into Phabricator.
 
-1. To uninstall Sourcegraph on your Phabricator instance run `./uninstall.sh <path/to/phabricator/root>` inside your Phabricator instance.
-2. Restart Phabricator instance.
+1. [Update your Sourcegraph site configuration](https://about.sourcegraph.com/docs/config/) to allow scripts on your Phabricator instance to communicate with your Sourcegraph instance:
 
-## Requirements
+```
+{
+   // ...
+   "corsOrigin": "$PHABRICATOR_URL"
+   // ...
+}
+```
 
-The installation script requires PHP 5.5 or newer and `php-tokenizer` (a default PHP extension).
+2. Update the `sourcegraph.url` configuration value to the URL where your Sourcegraph instance is hosted via the Phabricator UI or via the command line.
```

Comment:
Simplify this sentence: 

> 2. Set the Phabricator `sourcegraph.url` configuration value to your Sourcegraph URL.

And you can remove the `./bin/config` line because it is unnecessary (presumably they know how to configure Phab, and by including it, you imply that it is a locked config value that can only be set by the CLI). (Also, the `./` prefix is unnecessary.)